go.mod

```diff
@@ -10,7 +10,10 @@ require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	google.golang.org/grpc v1.27.0
+	k8s.io/api v0.19.4
 	k8s.io/apimachinery v0.19.4
+	k8s.io/client-go v0.19.4
+	k8s.io/klog v1.0.0
```

Comment:
where is this v1 used, lets use the `k8s.io/klog/v2` instead

Reply:
fixed